openlibrary/plugins/openlibrary/js/cover_add_manage.js

```diff
@@ -8,10 +8,6 @@ export function initAddCoverImport () {
         event.preventDefault();
     }
 
-    // page may not be loaded via iframe
-    if (parent && parent.closeThrobber) {
```

Comment:
Why does this get removed? (just wondering if this was intentional)

Reply:
For this one, I was going to figure out how to get it working in the context of not having a "parent" object from which to call a function (which I'm pretty sure was the case here). However, I couldn't find an element with class `throbber` to be operated on, so I just removed it thinking it wasn't necessary anymore.